### ISSUE TITLE
Template: Search button doesn't have proper text

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#66]
URL: Most
User Agent: Chrome/Firefox
Language: EN/FR

Issue Description: Search button from the template isn't done correctly.  When styles are turned off, the button has no label.
Steps to reproduce:
in Firefox, from the Web developer toolbar, turn disable all styles from the CSS tab.
navigate the page until you reach the search.
![image](https://user-images.githubusercontent.com/65728700/129082674-4595acfc-e70f-47c2-9ede-e71ca373dd8a.png)

Aria-label should also be in FR on the FR page. 
![image](https://user-images.githubusercontent.com/65728700/129083012-b30f69b7-19a3-49f4-a337-00f3f1af1a85.png)

Expected results: Button should have a proper label. 
Please follow canada.ca template
https://www.canada.ca/en.html

![image](https://user-images.githubusercontent.com/65728700/129083164-d457841b-f3d0-4c9e-b36c-8e328545ceb2.png)

**Comment by ndolter** [comment]
1. The button now has a label in both French and English
2. The aria label is now hidden
